ca/ca.go

```diff
@@ -764,6 +764,7 @@ func (ca *CertificateAuthorityImpl) OrphanIntegrationLoop() {
 				time.Sleep(time.Minute)
 				continue
 			}
+			time.Sleep(time.Second)
```

Comment:
nit: probably want to sleep after logging the error, rather than before, not super important though.